src/openforms/js/components/admin/form_design/logic/actions/dmn/VariableMapping.js

```diff
@@ -0,0 +1,105 @@
+import {FieldArray, useFormikContext} from 'formik';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+
+import DeleteIcon from 'components/admin/DeleteIcon';
+import Variable from 'components/admin/form_design/variables/types';
+import ButtonContainer from 'components/admin/forms/ButtonContainer';
+import Field from 'components/admin/forms/Field';
+import {TextInput} from 'components/admin/forms/Inputs';
+import Select from 'components/admin/forms/Select';
+
+import {inputValuesType} from './types';
+
+const VariableMapping = ({mappingName, values, formVariables}) => {
+  const intl = useIntl();
+  const {getFieldProps} = useFormikContext();
+
+  const confirmationMessage = intl.formatMessage({
+    description: 'Confirmation message to remove a mapping',
+    defaultMessage: 'Are you sure that you want to remove this mapping?',
+  });
+
+  return (
+    <FieldArray
+      name={mappingName}
+      render={arrayHelpers => (
+        <>
+          <table>
+            <thead>
+              <tr>
+                <th>
+                  <FormattedMessage
+                    defaultMessage="Open Forms variable"
+                    description="Open Forms variable label"
+                  />
+                </th>
+                <th>
+                  <FormattedMessage
+                    defaultMessage="DMN variable"
+                    description="DMN variable label"
+                  />
+                </th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody>
+              {values[mappingName].map((pair, index) => (
+                <tr key={index}>
+                  <td>
+                    <Field
+                      name={`${mappingName}.${index}.formVar`}
+                      htmlFor={`${mappingName}.${index}.formVar`}
+                    >
+                      <Select
+                        id={`${mappingName}.${index}.formVar`}
+                        name={`${mappingName}.${index}.formVar`}
+                        allowBlank={true}
+                        choices={formVariables.map(variable => [variable.key, variable.label])}
+                        {...getFieldProps(`${mappingName}.${index}.formVar`)}
```

Comment:
I think this already includes the `name` prop so you can probably remove it from line 57?